src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	private String blockNumber;
+
+	/**
+	 * Creates a Block object with the number given
+	 * @param number
+	 */
+	public Block(String number) {
+		this.setBlockNumber(number);
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)